wp-content/themes/core/components/document/document.twig

```diff
@@ -7,8 +7,7 @@
 
 		{{ do_action( 'tribe/body_opening_tag') }}
 
-		{# TODO: Move related a11y-skip-link css partial to this component #}
-		<a href="#a11y-skip-link-content" class="a11y-skip-link a11y-visual-hide">{{ __( 'Skip to main content' )|esc_html }}</a>
+		<a href="#a11y-skip-link-content" class="a11y-skip-link u-visual-hide">{{ __( 'Skip to main content' )|esc_html }}</a>
```

Comment:
When we get to a11y, we'll want add this as the first child item in site header/nav wrapper. Solves a "all things must be wrapped in a role based thing" issue.

Reply:
Good thought. I'll add a TODO there.